src/NSwag.MSBuild/NSwag.MSBuild.nuspec

```diff
@@ -27,11 +27,12 @@
     <file src="..\NSwag.Console.x86\bin\Release\net461\NSwag.x86.exe" target="tools/Win" />
     <file src="..\NSwag.Console.x86\bin\Release\net461\NSwag.x86.exe.config" target="tools/Win" />
 
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp1.0\Publish\**" target="tools/NetCore10" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp1.1\Publish\**" target="tools/NetCore11" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp2.0\Publish\**" target="tools/NetCore20" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp2.1\Publish\**" target="tools/NetCore21" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp2.2\Publish\**" target="tools/NetCore22" />
-    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp3.0\Publish\**" target="tools/NetCore30" />
+    <file src="..\NSwag.ConsoleCore\bin\release\netcoreapp1.0\**" target="tools/NetCore10" />
```

Comment:
Why no /Publish?

Reply:
oh yes sorry. When i  try to generate manually nuget package from Nswag.MsBuild it tells me nothing present in /Publish. To get my package quickly i remove /publish. Did not have time to investigate on this sorry.

Reply:
to get build.bat works

Reply:
strange. But the build is huge and requires some sdks etc

Reply:
strange. But the build is huge and requires some sdks etc

Reply:
strange. But the build is huge and requires some sdks etc

Reply:
strange. But the build is huge and requires some sdks etc

Reply:
Will check tomorrow

Reply:
i can cancel my changes on /publish if you want.

Reply:
Not sure but i think /publish is needed in order that this works correctly.